src/doc/rustc/src/platform-support/loongarch-linux.md

```diff
@@ -1,30 +1,24 @@
-# loongarch\*-unknown-linux-\*
+# `loongarch*-unknown-linux-*`
 
-**Tier: 2**
+[LoongArch][la-docs] Linux targets.
+LoongArch is a RISC ISA developed by Loongson Technology Corporation Limited.
 
-[LoongArch] is a new RISC ISA developed by Loongson Technology Corporation Limited.
+**Tier: 2 (with Host Tools)**
```

Comment:
we always put the tier on top, can you keep it there?

Reply:
Sure, will fix in a minute.

Reply:
BTW I referred to the `apple-darwin.md` and they put the target description above the tier levels; maybe that's because they're describing two targets at different tiers.